paths/locale_downloads/create.yaml

```diff
@@ -101,7 +108,6 @@ responses:
 x-code-samples:
   - lang: Curl
     source: |-
-
```

Comment:
sorry got removed with auto formatting, not sure if I should remove it or leave it. I think having the newline as code example should not be needed